scripts/download-sdks.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+cd "$(dirname $0)/../plugin-dev/Source/ThirdParty"
```

Comment:
We have to make sure that the `ThirdParty` directory exists first. Apparently, after removing all those binaries it has gone too

Reply:
```suggestion
mkdir -p "$(dirname $0)/../plugin-dev/Source/ThirdParty"
cd "$(dirname $0)/../plugin-dev/Source/ThirdParty"
```